### DISCUSSION TITLE
Automate the thread border router

### DISCUSSION THREAD

**Discussion by ghost** [openthread/openthread#10011] [Q&A] [answered]
Hello Sir,
can you help me to automate the thread border router? when I try to automate ifconfig up and thread start commands getting guru meditation errors.  I couldn't found  a possible method to solve it. I am using the esp32s3 and esp32h2 router boards. I have automated the ot_cli code and it works perfectly. Any support would be greatly appreciated.

Thanks you..

**Comment by chshu** [comment, marked as answer]
@niwayo787 Could you follow this instruction to enable the auto start mode: https://docs.espressif.com/projects/esp-thread-br/en/latest/dev-guide/build_and_run.html#wi-fi-based-thread-border-router, the BR will connect to the pre-configured Wi-Fi network and form a Thread network automatically.

**Reply by ghost** [reply]
Thank you for providing the instructions. I've followed the steps outlined in the documentation to enable auto-start mode for the Thread Border Router (BR). I've modified the configuration files as specified.

Thank you again for your guidance and support.
